src/index.html

```diff
@@ -11,6 +11,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="conteiner">
+      <div class="inputDiv" >
+        <img src="/images/Search.svg"
+        alt="search"
+        class="bigImg">
+        <input class="input"
+        data-qa="big"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;" >
+      </div>
+      <div class="inputDiv secondDiv">
+        <img src="/images/Search.svg"
+       class="smallImg"
+       alt="search">
+        <input class="input second"
+       data-qa="small"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;" >
+      </div>
```

Comment:
что с линтером? Выровняй отступы
```suggestion
      <div class="inputDiv" >
        <img 
          src="/images/Search.svg"
          alt="search"
          class="bigImg"
        >
        <input 
          class="input"
        data-qa="big"
        type="text"
        placeholder="Try &quot;Los Angeles&quot;" >
      </div>
      <div class="inputDiv secondDiv">
        <img src="/images/Search.svg"
       class="smallImg"
       alt="search">
        <input class="input second"
       data-qa="small"
        type="text"
        placeholder="Try &quot;Los Angeles&quot;" >
      </div>
```